schemas/iso19139/src/main/java/org/fao/geonet/schema/iso19139/ISO19139Namespaces.java

```diff
@@ -47,4 +47,7 @@
         Namespace.getNamespace("gmx", "http://www.isotc211.org/2005/gmx");
     public static final Namespace XLINK =
         Namespace.getNamespace("xlink", "http://www.w3.org/1999/xlink");
+    public static final Namespace GEONET =
```

Comment:
This code doesn't seem used, is required to add in ISO19139 schema this namespace?